tests/unit/src/utils.spec.js

```diff
@@ -49,7 +49,7 @@ describe('calculateTimeAgo', () => {
   it('should end with "ago" copy', () => {
     const date = new Date()
 
-    date.setSeconds(date.getSeconds() - 4)
+    date.setUTCSeconds(date.getUTCSeconds() - 4)
```

Comment:
This doesn't need to be changed, it's just a test that checks if dates from the past are displayed as "ago"